test/index.test.js

```diff
@@ -4,12 +4,11 @@ import { shallowToJson } from 'enzyme-to-json'
 import Counter from '../src/'
 import CounterSegment from '../src/CounterSegment'
 
-beforeAll(function () {
-  console.error = jest.fn(() => null) // disable React PropTypes warnings
-})
-
 describe('initialization', function () {
   it('throws an error when options are provided incorrectly', function () {
+    const _error = console.error
```

Comment:
dont sure that `_error` variable name would not cause confusion with unused variables nameing style. Why not `error`?